common/lib/xmodule/xmodule/templates/problem/imageresponse.yaml

```diff
@@ -4,19 +4,33 @@ metadata:
     markdown: !!null
```

Comment:
Thanks, @cptvitamin! Do you want us to open a ticket, or would you like to have the honors?